README.md

```diff
@@ -188,113 +252,166 @@ The following CSS:
+	.box-foo .box-bar {
+		width: 300px;
+	}
 }
 ```
 
+### `calc()`
+
+It also pairs well with [`postcss-calc`](https://github.com/postcss/postcss-calc) to reduce `calc()` expressions arising from using variables.
+
+```js
+var postcss = require('postcss');
+var cssvariables = require('postcss-css-variables');
+var calc = require('postcss-calc');
+
+var fs = require('fs');
+
+var mycss = fs.readFileSync('input.css', 'utf8');
 
+var output = postcss([
+		// Process any CSS variables
+		cssvariables(/*options*/)
+		// Then reduce `calc()` expressions
+		calc()
+	])
+	.process(mycss)
+	.css;
+
+console.log(output);
+```
+
+```css
+:root {
+	--page-margin: 1em;
+}
 
-## Interoperability
+/* Easy centering */
+.wrapper {
+	width: calc(100% - 2 * var(--page-margin));
+	margin-left: var(--page-margin);
+	margin-right: var(--page-margin);
+}
+```
 
-`postcss-css-variables` plays really nice with [`postcss-nested`](https://github.com/postcss/postcss-nested) in order to get a larger subset of CSS variables features. *See [Nested rules, Usage section](#nested-rules)*
+This will be transformed to:
 
-If you are using [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties) previously, we have a compatible feature set and more so you can switch over without having to refactor any of your code. You can just start writing the new awesome stuff.
+```css
+.wrapper {
+	width: calc(100% - 2em);
+	margin-left: 1em;
+	margin-right: 1em;
+}
+```
 
 
 # Why?
 
 This plugin was spawned out of a [discussion on the `cssnext` repo](https://github.com/cssnext/cssnext/issues/99) and a personal need.
 
-There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), which at the moment, covers less of the [CSS-variable/custom-property draft/spec](http://dev.w3.org/csswg/css-variables/) compared to this plugin. It also happened to not cover the feature I was after and after dealing with it, making an incomplete prototype piggybacking off of it; I decided to write my own starting from scratch. I do use some of the same/similar unit tests because that just makes sense to get proper code coverage.
+There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), although it restricts itself much more than this, preferring spec conformance over imperfect feature support.
+
+### Interoperability
+
+If you are/were already using [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), this should work out of the box, without issue, and other than differences in the JavaScript API, it's a drop-in replacement. The only difference in CSS handling is that this attempts much broader support, and you can just start taking advantage immediately. Note the [caveats](#caveats), in that although this does correctly support what `postcss-custom-properties` does, there are certain edge cases it doesn't get perfectly.
```

Comment:
Could you elaborate on what particular points you're getting at here? I'm somewhat confused.